lib/tasks/data_migrations/11_sync_watchlist_items_with_signals_telegram_subscriptions.rake

```diff
@@ -0,0 +1,43 @@
+namespace :data_migrations do
+  desc <<~DESC
+      Sync the WatchlistItems so that they reflect the current state of SignalsTelegramSubscriptions
+    DESC
+  task :sync_watchlist_items_with_signals_telegram_subscriptions => :environment do
+    ActiveRecord::Base.transaction do
+      SignalsTelegramUser.find_each do |signals_telegram_user|
+        watchlist = signals_telegram_user.user.watchlist
+        subscribed_coins = signals_telegram_user.subscribed_coins
+        watchlist_coins = watchlist.coins
+
+        watchlist_coins_to_add = subscribed_coins.select do |subscribed_coin|
+          watched = watchlist_coins.any? { |watchlist_coin| watchlist_coin.id == subscribed_coin.id }
+          next !watched
```

Comment:
Should this be `next if !watched` ?

Reply:
Thanks for the offline explanation!

Reply:
For future reference https://stackoverflow.com/a/11026130